protobuf/plugin/deepcopy/test/deepcopypb_test.go

```diff
@@ -1027,38 +1039,23 @@ func TestOneOfCopy(t *testing.T) {
 	if !in.Equal(out) {
 		t.Fatalf("%#v != %#v", in, out)
 	}
-	if &in.Fields == &out.Fields {
```

Comment:
We do use `oneof` in some places.

These non-message comparisons have been removed because they were redundant. The old generated code was comparing the same parent interface several times in a row.